AI/MIGraphX/Quantization/Readme.md

```diff
@@ -0,0 +1,50 @@
+
+# MIGraphX - Torch Examples
+
+# Summary
+The examples in this subdirectory showcase the functionality of the [MIGraphX](https://github.com/ROCmSoftwarePlatform/torch_migraphx/tree/master) library. The examples build on Linux for the ROCm (AMD GPU) backend.
```

Comment:
For the link please change text to Torch-MIGraphX from just MIGraphX. Thats confusing because MIGraphX is its own repo (not the one we are pointing to in this case)